### PR TITLE
manifest: Update sdk-zephyr revision

### PR DESCRIPTION
**Pull request by mkapala-nordic**
Change updates manifest to pull Bluetooth host new bt_conn_set_bondable() function which allows to set bondable flag per connection.

Jira: NCSDK-21190

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 35510c3d8e95826803ffaee1bb678163e1846320
+      revision: pull/1230/head
```
Comment (rlubos) [inline west.yml:62]: `9a0b8a317a91089f048c38233635240f21ab298d`

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
We have two concurrent PRs updating the manifest, I suggest we focus on https://github.com/nrfconnect/sdk-nrf/pull/11735 (which updates to the newest SHA) instead of rerrunning the CI on both.

**Comment by mkapala-nordic** [comment]
> We have two concurrent PRs updating the manifest, I suggest we focus on #11735 (which updates to the newest SHA) instead of rerrunning the CI on both.

Ok, I'll close this PR once mentioned PR is merged.

**Comment by mkapala-nordic** [comment]
Closing as sdk-zephyr revision is already newer

**Pull request closed without merging**